src/context/UserContext.tsx

```diff
@@ -23,8 +22,8 @@ export const UserProvider: React.FC<{ children: React.ReactNode }> = ({
 
   useEffect(() => {
     if (isAuthenticated) {
-      const bearer: any = getToken(TokenType.bearer);
-      const { email, picture, name }: any = jwtDecode(bearer);
+      const idToken: any = authManager.getToken.idToken() || "";
+      const { picture, name, email }: any = jwtDecode(idToken);
```

Comment:
I would move this logic to AuthManager - new function which returns current user.